km/km_snapshot.c

```diff
@@ -529,10 +545,30 @@ int km_snapshot_block(km_vcpu_t* vcpu)
    in_snapshot = 1;
    pthread_mutex_unlock(&snap_mutex);
    km_vcpu_pause_all(vcpu, ALL);   // Wait for everyone to get to the pause point.
+
+   for (int i = 0; i < KVM_MAX_VCPUS; i++) {
+      if (machine.vm_vcpus[i] == NULL) {
+         continue;
```

Comment:
Could we put a break statement here?
There should be no more vcpu's after this.

Reply:
is that true?

Reply:
ok